scripts/split-e2e-tests.ts

```diff
@@ -85,6 +85,22 @@ const WINDOWS_TEST_ALLOWLIST: string[] = [
   'auth_5f_pkg',
 ];
 
+// some tests may require a larger executor, specify those here
+const USE_LINUX_LARGE_VM: string[] = [
```

Comment:
verbs are usually reserved for functions/methods.

```suggestion
const JOBS_RUNNING_ON_LINUX_LARGE_VM: string[] = [
```

Reply:
done!